apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/JvmTaskConfigurator.kt

```diff
@@ -40,9 +39,9 @@ object JvmTaskConfigurator {
     }
 
     val sourceDirectorySet = if (!compilationUnit.generateKotlinModels()) {
```

Comment:
Little unrelated but I have a suggestion: Inverting this condition makes it hard to understand. I saw Kotlin in the else and confused because I didn't see the negation

Reply:
:+1: Indeed, I just swapped the branches.

Reply:
I love your commit message 😊